### PR TITLE
OpenAPI: cues_api

### PR DESCRIPTION
**Pull request by markxiong0122**

### CODE REVIEW

File: api/cues_api.py [hunk 1]
```diff
@@ -33,12 +35,14 @@ class CuesAPI(basehandlers.APIHandler):
 
   def do_post(self, **kwargs):
     """Dismisses a cue card for the signed in user."""
-    cue = self.get_param('cue', allowed=ALLOWED_CUES)
```
Comment (DanielRyanSmith) [inline api/cues_api.py:36]: The `allowed` param is no longer being used to abort in the context of receiving disallowed values for cue. We probably still want to use this helper method for obtaining the value from the request.

Comment (markxiong0122) [reply api/cues_api.py:36]: I obtained values the request by casting the input to the generated model in new line 31 (and it also has enum to limit the choice to only ALLOWED_CUES)

Comment (DanielRyanSmith) [reply api/cues_api.py:36]: What is the behavior if a request is sent with a bad value? Currently, the request will be aborted with a 400 status code and this mesage:

https://github.com/GoogleChrome/chromium-dashboard/blob/796b60b4232ba5e0dee0bbcd1b01583f5609e0d1/framework/basehandlers.py#L106-L107

Comment (markxiong0122) [reply api/cues_api.py:36]: it handled with an extra condition check, which throws a 400 error if it's not allowed as we discussed yesterday

Comment (DanielRyanSmith) [reply api/cues_api.py:36]: Perfect, thanks!

**Review by DanielRyanSmith — APPROVED** [review]

**Pull request merged**